### PR TITLE
fix: updated Obo image version in CSV

### PR DESCRIPTION
**Pull request by vimalk78**
fix: updated Obo image version in CSV
fix: increase timeout for installing OLM on kind cluster

fixes: #273

### CODE REVIEW

File: hack/kind/setup.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ label_infra_node() {
 
 setup_olm() {
   header "Install OLM"
-  "$PROJECT_ROOT_DIR"/tmp/bin/operator-sdk olm install
```
Comment (sthaha) [inline hack/kind/setup.sh:96]: Unrelated change :), perhaps a separate PR 🙏

File: deploy/package-operator/operator/kustomization.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: Kustomization
 namespace: observability-operator
 images:
 - name: observability-operator
-  newTag: 0.0.17
```
Comment (sthaha) [inline deploy/package-operator/operator/kustomization.yaml:6]: when the version gets bumped to 0.0.21, would we have to do this manually? If so, can we automate this somehow in Makefile?

Comment (vimalk78) [reply deploy/package-operator/operator/kustomization.yaml:6]: updated

**Review by sthaha — APPROVED** [review]
